apps/backend/src/libs/constants/white-routes.constant.ts

```diff
@@ -5,6 +5,20 @@ import { type WhiteRoute } from "./libs/types/types.js";
 const WHITE_ROUTES: WhiteRoute[] = [
 	{ method: "POST", path: `${APIPath.AUTH}${AuthApiPath.SIGN_UP}` },
 	{ method: "POST", path: `${APIPath.AUTH}${AuthApiPath.SIGN_IN}` },
+
+	{ method: "GET", path: "/documentation" },
```

Comment:
Tbh I don't like that you list all routes one by one. Can we do smth like `/documentation/*`?

Reply:
Let's use the solution with regex. The current solution is not safe for future possible swagger changes